### PR TITLE
Upstream provider induced ice cream

### PR DESCRIPTION
**Pull request by woeldiche**
Due to a recent number of issues caused fully or partially by our upstream service providers and the exemplary and always tireless response from our on-duty teams (with a suitable amount of snark included of course) I suggest that we expand the cake legislation to cover this incident.

### CODE REVIEW

File: legislation.tex [hunk 1]
```diff
@@ -86,5 +86,6 @@
         \item Loosing office key fob.
         \item Occupying a meeting room booked by another person (without leaving immediately).
         \item If network down > 15 min, or network down > 2 times on one day (calendar days, not 24 hours) POP2 will provide cake.
+        \item Incident with a duration longer than 8 hours or requiring action from Clearhaus employees on duty, fully or primarily caused by upstream service provider (\emph{cake responsible organization} is Clearhaus A/S. The preferred type of cake for this rule is icecream).
```
Comment (woeldiche) [inline legislation.tex]: This should be `requiring action from Clearhaus employees on duty outside of normal office hours`

Comment (ct-clearhaus) [reply legislation.tex]: > normal office hours

That can be very hard to define and identify 😉

Comment (woeldiche) [reply legislation.tex]: Yes. That is why it's important with a review process for all important legislation. This is the 'høringsperiode' for this particular change.

Comment (jsmpereira) [reply legislation.tex]: It's never only _on duty_ :sweat_smile:

**Review by ct-clearhaus — DISMISSED** [review]

File: legislation.tex [hunk 2]
```diff
@@ -86,5 +86,6 @@
         \item Loosing office key fob.
         \item Occupying a meeting room booked by another person (without leaving immediately).
         \item If network down > 15 min, or network down > 2 times on one day (calendar days, not 24 hours) POP2 will provide cake.
+        \item Incident with a duration longer than 8 hours or requiring action from Clearhaus employees on duty outside of normal office hours, fully or primarily caused by upstream service provider (\emph{cake responsible organization} is Clearhaus A/S. The preferred type of cake for this rule is icecream).
```
Comment (ct-clearhaus) [inline legislation.tex]:
```suggestion
        \item Incident with a duration longer than 8 hours or requiring action from Clearhaus employees on duty outside of normal office hours, fully or primarily caused by upstream service provider (\emph{cake responsible organization} is Clearhaus A/S. The preferred type of cake for this rule is ice cream).
```

Is ice cream preferred no matter the time of year? 🤔

Comment (jsmpereira) [reply legislation.tex]: Of course. Ice-cream is totally decoupled from time

**Review by brorbw — APPROVED** [review]

**Comment by ct-clearhaus** [comment]
Thanks @cbobach, I've now fixed that:
<kbd>
Screenshot 2023-10-25 at 08 22 38
</kbd>

**Pull request merged**